internal/cluster/validations/validations.go

```diff
@@ -40,7 +40,6 @@ const (
 
 // PullSecretValidator is used run validations on a provided pull secret
 // it verifies the format of the pull secrete and access to required image registries
-// go:generate mockgen -source=validations.go -package=validations -destination=mock_validations.go
```

Comment:
Why has this been removed?

Reply:
we don't need mock_validation.go now as it's not called anywhere, so I removed it

Reply:
@rewantsoni , I guess this interface refers to https://github.com/openshift/assisted-service/blob/master/internal/cluster/validations/mock_validations.go and has nothing to do with OCS validations. Please restore the changes again

Reply:
fixed